dojo-chess/src/utils.cairo

```diff
@@ -0,0 +1,48 @@
+use dojo_chess::models::PieceType;
+use starknet::ContractAddress;
+
+fn is_piece_is_mine(maybe_piece: PieceType) -> bool {
+    false
+}
+
+fn is_correct_turn(maybe_piece: PieceType, caller: ContractAddress, game_id: felt252) -> bool {
+    true
+}
+
+fn is_out_of_board(next_position: (u32, u32)) -> bool {
+    let (n_x, n_y) = next_position;
+    if n_x > 7 || n_x < 0 {
```

Comment:
Could it be sufficient?

```
fn is_out_of_board(next_position: (u32, u32)) -> bool {
    let (n_x, n_y) = next_position;
    n_x < 8 && n_y < 8
}
```

Note: I don't think `n_x/y < 0` can be verified

Reply:
1. What more rules could we add?
2. Right, since `n_x/y` is `u32` type, no need to check `<0`